test/ui_test_suite.rb

```diff
@@ -1361,6 +1361,16 @@ def teardown
     assert primary_data_count == 1, "did not find correct number of primary data files, expected 1 but found #{primary_data_count}"
     assert other_data_count == 19, "did not find correct number of other data files, expected 19 but found #{primary_data_count}"
 
+    # give time for parses to complete
```

Comment:
For easier reading:

```suggestion
    # give 2 minutes time for parses to complete
```

I like that this serves not only as a functional test, but also a performance test.

Reply:
[ad9f42c](https://github.com/broadinstitute/single_cell_portal_core/pull/356/commits/ad9f42cf41c378362223edbf639e821c2eb265b2)